packages/strapi-plugin-users-permissions/admin/src/translations/ru.json

```diff
@@ -11,7 +11,7 @@
   "Auth.form.error.confirmed": "Адрес электронной почты не подтвержден.",
   "Auth.form.error.email.invalid": "Неправильный адрес электронной почты.",
   "Auth.form.error.email.provide": "Укажите свое имя пользователя или адрес электронной почты.",
-  "Auth.form.error.email.taken": "Почтовый адрес уже используется",
+  "Auth.form.error.email.taken": "Данный Email уже используется",
```

Comment:
Why everywhere "email" with small letter, but there with a capital?

Reply:
@chiliec 
Hello, this is my fault. I’ll update now PR